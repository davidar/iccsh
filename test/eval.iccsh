:declare x int x
:declare y int y
x = 4
:printf %d x
y = 2
:printf %d y

:eval
x ^= y;
y ^= x;
x ^= y;

:printf %d,%d x, y

:declare i int i
for(i = 0; i < 10; i++) \
    printf("%d\n", i);
