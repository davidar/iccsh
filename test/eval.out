4
2
2,4
0
1
2
3
4
5
6
7
8
9
